pom.xml

```diff
@@ -83,6 +83,7 @@
                     <runtime>
                         <!-- runtime os, could be windows, linux or docker-->
                         <os>windows</os>
+                        <javaVersion>8</javaVersion>
                         <!-- for docker function, please set the following parameters -->
```

Comment:
better to provide same version with gradle

Reply:
Yes, updated it to java 8